compiler/optimizations/gpuTransforms.cpp

```diff
@@ -1567,19 +1672,32 @@ void GpuKernel::processGpuPrimitivesBlock(BlockStmt* block) {
               call->numActuals() == 2) {
             auto sym1 = toSymExpr(blockSizeCall_->get(2))->symbol();
             auto sym2 = toSymExpr(call->get(2))->symbol();
-
             if (sym1 == sym2) continue;
           }
-
           USR_FATAL(call, "Can only set GPU block size once per GPU-eligible loop.");
         }
+
         blockSizeCall_ = call;
         blockSize_ = toSymExpr(call->get(1))->symbol();
       }
-    }
+      else if (call->isPrimitive(PRIM_GPU_SET_ITERS_PER_THREAD)) {
+        if (itersPerThreadCall_ != nullptr) {
+          // See the comment above for PRIM_GPU_SET_BLOCKSIZE
+          if (itersPerThreadCall_->numActuals() == 2 &&
+              call->numActuals() == 2) {
+            auto sym1 = toSymExpr(itersPerThreadCall_->get(2))->symbol();
+            auto sym2 = toSymExpr(call->get(2))->symbol();
+            if (sym1 == sym2) continue;
+          }
```

Comment:
I can do it. Or we can wait until somebody touches this code again and do it at that time. :)

Seriously speaking, I did not do it because I was not confident that going forward the two attributes will remain similar in this regard.